### ISSUE TITLE
Build with /guard:cf

### ISSUE THREAD

**Issue by asvc** [picotorrent/picotorrent#833]
Picotorrent seems to be already built with `/DYNAMICBASE`, `/HIGHENTROPYVA` and `/NXCOMPAT`:

```powershell
Get-PESecurity -File 'C:\Program Files\PicoTorrent\PicoTorrent.exe'                                                       

ARCH             : AMD64
DotNET           : False
ASLR             : True
DEP              : True
Authenticode     : False
StrongNaming     : N/A
SafeSEH          : N/A
ControlFlowGuard : False
HighentropyVA    : True
```

Would it be possible to add [`/guard:cf`](https://docs.microsoft.com/en-us/cpp/build/reference/guard-enable-control-flow-guard?view=vs-2019) for the full roster of mitigations?

**Comment by vktr** [comment]
Hi! Sounds reasonable. I'll add it as a feature request 😃

**Comment by vktr** [comment]
I've added this to the release builds for 0.19.

**Comment by vktr** [comment]
This was fixed in #843 and will be released soon.
